client/package.json

```diff
@@ -144,5 +144,6 @@
   },
   "browserslist": [
     "defaults"
-  ]
+  ],
+  "packageManager": "yarn@1.22.19"
```

Comment:
What's the rationale for this? Is it necessary?

Reply:
🤦🏻 Not at all. My setup uses Yarn v3 and to avoid changing the lockfile formatting, I ran `yarn set-version classic` - which introduced a bunch of random changes throughout the repo - I thought I had caught them all. Turns out it's not super intuitive to run a separate version of Yarn in a specific workspace. I'll uncommit this and watch for it again!